docs/api.md

```diff
@@ -10,180 +10,152 @@
   - [`tx`](#tx)
-      - [Arguments](#arguments-18)
-      - [Returns](#returns-13)
-      - [Usage](#usage-26)
-      - [Examples](#examples-7)
+    - [Arguments](#arguments-18)
+    - [Returns](#returns-13)
+    - [Usage](#usage-26)
+    - [Examples](#examples-7)
     - [Event Polling Utility](#event-polling-utility)
   - [`events`](#events)
-      - [Arguments](#arguments-19)
-      - [Returns](#returns-14)
-      - [Usage](#usage-27)
-      - [Examples](#examples-8)
-- [Types, Interfaces, and Definitions](#types-interfaces-and-definitions)
-    - [`Builders`](#builders-1)
-    - [`Interaction`](#interaction)
-    - [`CurrentUserObject`](#currentuserobject)
-    - [`AuthorizationObject`](#authorizationobject)
-    - [`SignableObject`](#signableobject)
-    - [`AccountObject`](#accountobject)
-    - [`Address`](#address)
-    - [`ArgumentObject`](#argumentobject)
-    - [`ArgumentFunction`](#argumentfunction)
-    - [`Authorization Function`](#authorization-function)
-      - [Usage](#usage-28)
-      - [Examples:](#examples-9)
-    - [`Signing Function`](#signing-function)
-      - [Payload](#payload)
-      - [Usage](#usage-29)
-      - [Examples:](#examples-10)
-    - [`TransactionRolesObject`](#transactionrolesobject)
-    - [`EventName`](#eventname)
-    - [`Contract`](#contract)
-    - [`KeyObject`](#keyobject)
-    - [`BlockObject`](#blockobject)
-    - [`BlockHeaderObject`](#blockheaderobject)
-    - [`CollectionGuaranteeObject`](#collectionguaranteeobject)
-    - [`CollectionObject`](#collectionobject)
-    - [`ResponseObject`](#responseobject)
-    - [`Event Object`](#event-object)
-    - [`Transaction Statuses`](#transaction-statuses)
-    - [`GRPC Statuses`](#grpc-statuses)
-    - [`FType`](#ftype)
+    - [Arguments](#arguments-19)
+    - [Returns](#returns-14)
+    - [Usage](#usage-27)
+    - [Examples](#examples-8)
+- [Types, Interfaces, and Definitions](#types-interfaces-and-definitions) - [`Builders`](#builders-1) - [`Interaction`](#interaction) - [`CurrentUserObject`](#currentuserobject) - [`AuthorizationObject`](#authorizationobject) - [`SignableObject`](#signableobject) - [`AccountObject`](#accountobject) - [`Address`](#address) - [`ArgumentObject`](#argumentobject) - [`ArgumentFunction`](#argumentfunction) - [`Authorization Function`](#authorization-function) - [Usage](#usage-28) - [Examples:](#examples-9) - [`Signing Function`](#signing-function) - [Payload](#payload) - [Usage](#usage-29) - [Examples:](#examples-10) - [`TransactionRolesObject`](#transactionrolesobject) - [`EventName`](#eventname) - [`Contract`](#contract) - [`KeyObject`](#keyobject) - [`BlockObject`](#blockobject) - [`BlockHeaderObject`](#blockheaderobject) - [`CollectionGuaranteeObject`](#collectionguaranteeobject) - [`CollectionObject`](#collectionobject) - [`ResponseObject`](#responseobject) - [`Event Object`](#event-object) - [`Transaction Statuses`](#transaction-statuses) - [`GRPC Statuses`](#grpc-statuses) - [`FType`](#ftype)
```

Comment:
The types here aren't formatting correctly - showing as one big block rather than by bullet.